### DISCUSSION TITLE
[835960] The Talos Principle 2  塔罗斯的法则2  仅本体

### DISCUSSION THREAD

**Discussion by bearbiscuit-0V0** [pjy612/SteamManifestCache#40] [Share]
[835960_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13685671/835960_config.vdf.txt)
appid:835960
keys:
835961;fb50a63e9de1e5f24f3cfb6de54ba276dff93be8dd3e80dc9f293a0aa862db6d

**Comment by bearbiscuit-0V0** [comment]
code:
0A394B25083DECC41E4619AE97CBD4D744B829A6

**Reply by pjy612** [reply]
How do I find you?

**Reply by bearbiscuit-0V0** [reply]
email:
passls@vip.qq.com

**Reply by pjy612** [reply]
查收
